src/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+// const { default: axios } = require("axios");
+
+const DEFAULT_CITY = 'Seattle';
+// Select the HTML Element the event will occur on
+const increaseTempElement = document.getElementById('increaseTempBtn');
+const decreaseTempElement = document.getElementById('decreaseTempBtn');
+const currentTempTag = document.getElementById('currentTemp');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const cityNameReset = document.getElementById('cityNameReset');
+const SkyDropDown = document.getElementById('SkyChoice');
+const realtimeTemp = document.getElementById('realTempBtn');
+
+const changingTempColorAndLandscape = () => {
+  let currentTemp = parseInt(currentTempTag.textContent);
+  if (currentTemp >= 80) {
+    currentTempTag.className = 'red';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (currentTemp >= 70) {
```

Comment:
Nice solution here. How might you refactor this so the logic is more data driven? Here you have several if/else if statements. 

What if you had an object with the keys as the type of weather and the values as the corresponding display? Then you could iterate over the object to set landscape.textContext?